### PR TITLE
Stream API HW

### PR DESCRIPTION
**Pull request by ydav-072**

### CODE REVIEW

File: src/ru/freeIt/homework/streamAPI/task2/Query2.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        File file = new File("src/ru/freeIt/homework/streamAPI/task2/resource/string.txt");
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new FileReader(file));
+            StringBuilder stringBuilder = new StringBuilder();
+            while (bufferedReader.ready()) {
+                stringBuilder.append(bufferedReader.readLine());
+            }
+            bufferedReader.close();
+            String[] stringList = stringBuilder.toString().replaceAll("\\p{P}", "").split(" ");
+            Arrays.stream(stringList).sorted(String::compareToIgnoreCase).forEach(System.out::println);
```
Comment (atlamax) [inline src/ru/freeIt/homework/streamAPI/task2/Query2.java:19]: Try this:
Comparator.naturalOrder()

File: src/ru/freeIt/homework/streamAPI/task2/Query2.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            System.out.println();
+            Arrays.stream(stringList).sorted((el1, el2) -> el2.compareToIgnoreCase(el1)).forEach(System.out::println);
```
Comment (atlamax) [inline src/ru/freeIt/homework/streamAPI/task2/Query2.java:21]: Try this:
Comparator.reverseOrder()

**Comment by atlamax** [comment]
@ydav-072 
In general it's approved. Merge by yourself after fixes in Query2.

File: src/ru/freeIt/homework/streamAPI/task2/Query2.java (hunk 2 above)
Comment (ydav-072) [reply src/ru/freeIt/homework/streamAPI/task2/Query2.java:21]: sorry, but main idea it's using StreamAPI and lambda and I use it. Ok, I will change that

File: src/ru/freeIt/homework/streamAPI/task2/Query2.java (hunk 1 above)
Comment (ydav-072) [reply src/ru/freeIt/homework/streamAPI/task2/Query2.java:19]: sorry, but main idea it's using StreamAPI and lambda and I use it. Ok, I will change that

**Comment by ydav-072** [comment]
Thank you!) It's very nessery for me

File: src/ru/freeIt/homework/streamAPI/task2/Query2.java (hunk 1 above)
Comment (atlamax) [reply src/ru/freeIt/homework/streamAPI/task2/Query2.java:19]: You will use it. Final result should be:

```
Arrays.stream(stringList)
  .sorted(Comparator::naturalOrder)
  .forEach(System.out::println);
```

**Pull request merged**